From: Povilas Kanapickas <povilas@radix.lt>
Date: Thu, 16 Mar 2017 12:04:41 +0300
Subject: do-not-distribute-fonts

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1e4948e..9977f3b 100644
--- a/Makefile
+++ b/Makefile
@@ -80,7 +80,7 @@ dist: clean
 install: all
 	# install the devhelp documentation
 	pushd "output/reference" > /dev/null; \
-	find . -type f \
+	find . -type f -not -iname "*.ttf" \
 		-exec install -DT -m 644 '{}' "$(DESTDIR)$(docdir)/html/{}" \; ; \
 	popd > /dev/null
 
